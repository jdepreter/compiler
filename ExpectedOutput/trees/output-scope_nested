digraph {
	1 [label=C]
	3 [label=scope]
	1 -> 3
	5 [label=definition]
	3 -> 5
	6 [label=int]
	5 -> 6
	7 [label=assignment2]
	5 -> 7
	8 [label=c]
	7 -> 8
	16 [label=1]
	7 -> 16
	18 [label=scope]
	3 -> 18
	20 [label=definition]
	18 -> 20
	21 [label=int]
	20 -> 21
	22 [label=assignment2]
	20 -> 22
	23 [label=c]
	22 -> 23
	31 [label=1]
	22 -> 31
	33 [label=assignment]
	18 -> 33
	34 [label=c]
	33 -> 34
	43 [label="+"]
	33 -> 43
	42 [label=c]
	43 -> 42
	48 [label=1]
	43 -> 48
	50 [label=method_call]
	18 -> 50
	51 [label=printf]
	50 -> 51
	60 [label=c]
	50 -> 60
	61 [label="empty arg"]
	50 -> 61
	63 [label=assignment]
	3 -> 63
	64 [label=c]
	63 -> 64
	73 [label="+"]
	63 -> 73
	72 [label=c]
	73 -> 72
	78 [label=2]
	73 -> 78
	80 [label=method_call]
	3 -> 80
	81 [label=printf]
	80 -> 81
	90 [label=c]
	80 -> 90
	91 [label="empty arg"]
	80 -> 91
}
