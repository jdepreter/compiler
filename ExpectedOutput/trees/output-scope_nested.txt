digraph {
	1 [label=C]
	3 [label=scope]
	1 -> 3
	5 [label=definition]
	3 -> 5
	6 [label=int]
	5 -> 6
	7 [label=assignment2]
	5 -> 7
	8 [label=c]
	7 -> 8
	16 [label=1]
	7 -> 16
	18 [label=scope]
	3 -> 18
	20 [label=definition]
	18 -> 20
	21 [label=int]
	20 -> 21
	22 [label=assignment2]
	20 -> 22
	23 [label=c]
	22 -> 23
	31 [label=1]
	22 -> 31
	33 [label=assignment]
	18 -> 33
	34 [label=c]
	33 -> 34
	43 [label="+"]
	33 -> 43
	42 [label=c]
	43 -> 42
	48 [label=1]
	43 -> 48
	50 [label=assignment]
	3 -> 50
	51 [label=c]
	50 -> 51
	60 [label="+"]
	50 -> 60
	59 [label=c]
	60 -> 59
	65 [label=2]
	60 -> 65
}
